scout/adapter/mongo/disease_terms.py

```diff
@@ -32,8 +34,9 @@ def query_disease(
         if query:
             query_dict = {
                 "$or": [
-                    {"disease_nr": {"$regex": query, "$options": "i"}},
-                    {"description": {"$regex": query, "$options": "i"}},
+                    {"disease_nr": {"$regex": query, REGEX_OPTIONS: REGEX_IGNORECASE}},
```

Comment:
Weird - the number is already there?

Reply:
Ah, right, as on chat - it's actually an `int32` in the db now. Then we could just drop this from the search. Or coerce it to search as str instead of searching in the `disease_nr`.